tests/certification/bindings/alicloud/nacos/go.mod

```diff
@@ -9,19 +9,19 @@ require (
 	github.com/dapr/go-sdk v1.4.0
 	github.com/dapr/kit v0.0.2-0.20210614175626-b9074b64d233
 	github.com/nacos-group/nacos-sdk-go/v2 v2.0.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.7.1
```

Comment:
Those file chanages are not relates to dubbo, can we revert them?

Reply:
Also please add certification tests.

Reply:
Fine, I would revert all un necessary change of gomod

Reply:
@daixiang0 The problem is that,  my dubbo pkg updates some version in go mod, and after running `make modtidy-all` the sub go mod in /tests are affected, and updated too.

Reply:
Ok, that's fine.